orca-igor/src/test/groovy/com/netflix/spinnaker/orca/igor/BuildServiceSpec.groovy

```diff
@@ -29,10 +29,11 @@ class BuildServiceSpec extends Specification {
   private static final JOB_NAME_ENCODED = "name/with/slashes%20and%20spaces"
   private static final PARAMS = ['key': 'value']
   private static final FILENAME = 'file.txt'
+  private static final QUEUE_BUILD = ''
```

Comment:
```suggestion
  private static final QUEUED_BUILD = ''
```
to match the method argument?  Also, could you make it a non-empty string to serve as an example?